app/pages/nonsipp/memberpayments/PensionCommencementLumpSumAmountPage.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.memberpayments
+
+import config.Refined.Max300
+import models.{Mode, PensionCommencementLumpSum}
+import models.SchemeId.Srn
+import pages.QuestionPage
+import utils.RefinedUtils.RefinedIntOps
+import play.api.libs.json.JsPath
+
+case class PensionCommencementLumpSumAmountPage(srn: Srn, index: Max300, mode: Mode)
+    extends QuestionPage[PensionCommencementLumpSum] {
+
+  override def path: JsPath = MemberPaymentsPage.path \ toString \ index.arrayIndex.toString
```

Comment:
Path would need to be membersPayments\memberDetails\memberLumpSumReceived\ toString, for reference: https://github.com/hmrc/pension-scheme-return-frontend/blob/6ef70f53c1b72f2f667d14eb84c82a008a7fd131/app/pages/nonsipp/receivetransfer/package.scala#L29
and 
https://github.com/hmrc/pension-scheme-return-frontend/blob/6ef70f53c1b72f2f667d14eb84c82a008a7fd131/app/pages/nonsipp/receivetransfer/TotalValueTransferPage.scala#L30